### ISSUE TITLE
[Fabric] Plugin Messages not working with Velocity or Lightfall

### ISSUE THREAD

**Issue by addstar** [IzzelAliz/Arclight#1248]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

arclight-1.20.4-1.0.3-SNAPSHOT-ba03152

### OS & Java versions

OpenJDK (build 17.0.10+7-Ubuntu-122.04.1)

### Plugins and Mods

```raw
Any Spigot plugin that tries to send or receive Plugin Messages via the Spigot API (`sendPluginMessage`).
```

### Description

Using Arclight-Fabric, I have been unable to make Plugin Messages (via `sendPluginMessage`) work between Arclight and other servers. I've used vPackerLogger on Velocity to troubleshoot and it looks like Plugin Messages are never received by Velocity but they work fine on other Spigot servers on my network. It seems like the messages are never sent from Arclight to the proxy. Likewise, _no_ Plugin Messages are received by Arclight from Velocity (or Lightfall).

I've tested with various "clean" Arclight-Fabric servers, several different plugins that send plugin messages and nothing seems to work. I can't make Plugin Messages work at all.

So basically it seems like Plugin Message communications are completely broken or maybe I just don't have it configured properly. Are you aware of any issues with this or do I need specific configuration/mods/plugins to make this work?

**Comment by addstar** [comment]
Thank you! It works great!! :partying_face:
